content/documentation/staging/quick-start/index.adoc

```diff
@@ -16,108 +16,59 @@ weight: 2
 :imagesdir: /images/quickinstall/
 :sectlinks:
 
-icon:lightbulb[size=1x]{nbsp} This page is intended for users installing Kiali for the first time, not recommended for custom production environments. In case of custom installations, visit the link:{{< ref "/documentation/staging/installation-guide" >}}[installation guide].
+icon:lightbulb[size=1x]{nbsp} This page is intended for users installing Kiali for the first time. These instructions are not recommended to install Kiali in production environments. In the case of custom installations, visit the link:{{< ref "/documentation/staging/installation-guide" >}}[installation guide].
 
-== Installing Kiali
+icon:lightbulb[size=1x]{nbsp} If you are installing on OpenShift, you must grant the `cluster-admin` role to the user that is installing Istio and Kiali. If OpenShift is installed locally on the machine you are using, the following command should log you in as user `system:admin` which has this `cluster-admin` role: `oc login -u system:admin`
```

Comment:
Please, OpenShift shoudn't be in the quickstart, it's a platform specific requeriment.

Reply:
well you and @jshaughn need to come to an agreement, because I added it as part of his review.

Reply:
QuickStart is not intended to OpenShift users.
Please, don't put it there.

OpenShift users have several ways to install Istio, to me install istio upstream in OpenShift is a corner case that a) it's described as a platform specific in Istio page b) should be also follow that format in the quick start.

QuickStart is for community users that want to focus on features.

Reply:
> QuickStart is not intended to OpenShift users.

I see no reason to ruin the quick start if the user is using an OS kubernetes env.  It's just one line.  Having said that, perhaps a decent compromise is to treat that OpenShift requirement as a re-requesite and move the OS note to that section, which is already mentioned in the quick start.

Reply:
It's a prerequesite and as commented, the better place is the installation guide.
Istio has more than a dozen of platform specifics:
![image](https://user-images.githubusercontent.com/1662329/89648252-3bb08a00-d8bf-11ea-8f10-8ca9f42a0ce6.png)

I'm okey to not specify them if we don't need to, but I don't think is good that it's marked in the Quick Start, it may create a biased perception that OpenShift is necessary (it had happen in the past).